docker/docker-entrypoint.sh

```diff
@@ -3,7 +3,12 @@
 # Parameters
 if [[ "${NAT_IP_AUTO}" == "true" ]]; then
   export CODEX_NAT=$(hostname --ip-address)
-  echo "Set CODEX_NAT: ${CODEX_NAT}"
+  echo "Internal: Set CODEX_NAT: ${CODEX_NAT}"
+fi
+
+if [[ "${NAT_PUBLIC_IP_AUTO}" == "true" ]]; then
+  export CODEX_NAT=$(curl https://ipinfo.io/ip)
```

Comment:
I don't know if we can rely on this, it's probably better to setup our own endpoint for that.